src/Login.java

```diff
@@ -1,17 +1,10 @@
 
-    import java.util.Scanner;
-
-    public class Login {
-
-
-        public void excute(String setName,String setPassword) {
- if (UserContainer.isNameExists(setName,setPassword)) {
-     System.out.println("\n" + "Successful entry");
- }
- else {
-     System.out.println("wrong username or password");
- }
+public class Login {
+    public void execute(String userName, String userPassword) {
+        if (UserContainer.isNameAndPasswordExisits(userName, userPassword)) {
```

Comment:
The same style everywhere 
`isNameAndPasswordExisits` -> `isUserNameAndPasswordExisits`